frontend/micro-ui/web/micro-ui-internals/packages/modules/Masters/src/configs/createOrganizationConfig.js

```diff
@@ -0,0 +1,318 @@
+import React from "react";
+import { useTranslation } from "react-i18next";
+
+export const createOrganizationConfig = () => {
+    const { t } = useTranslation()
+
+    const userInfo = Digit.UserService.getUser();
+    const tenantId = Digit.ULBService.getCurrentTenantId();
+    const ULB = Digit.Utils.pt.getCityLocale(tenantId);
+    const city = userInfo && userInfo?.info?.permanentCity;
+   
+    const {data: Localities } = Digit.Hooks.useLocation(tenantId, {}, 'Locality')
+    const {data: Wards } = Digit.Hooks.useLocation(tenantId, {}, 'Ward')
+    
+    const headerLocale = Digit.Utils.locale.getTransformedLocale(tenantId)
+    let localityOptions = []
+    Localities &&
```

Comment:
for formatting, any hook data send a function as default which will give u formatted response
ref : frontend/micro-ui/web/micro-ui-internals/packages/libraries/src/hooks/useCustomMDMS.js

Reply:
Passing formatted data from hook itself, please check

Reply:
i was saying about this part   
  let wardOptions = []
    Wards &&
        Wards?.TenantBoundary[0]?.boundary.map(item => {
            wardOptions.push({code: item.code, name: item.name,  i18nKey: t(`${headerLocale}_ADMIN_${item?.code}`) })
        })

Reply:
you can send. a function called default in that it can do formatting instead of doing after getting data